src/constants.js

```diff
@@ -216,3 +216,13 @@ export const TIMEOUT = 10000;
  * Default timeout for send tokens request in milliseconds
  */
 export const SEND_TOKENS_TIMEOUT = 300000;
+
+/**
+ * Number of iterations to execute when hashing the password
+ */
+export const HASH_ITERATIONS = 1000;
```

Comment:
In the first document you sent (the pdf), they suggest at least 1,000. 

> A minimum iteration count of 1,000 is recommended. For especially critical keys, or for very
powerful systems or systems where user-perceived performance is not critical, an
iteration count of 10,000,000 may be appropriate. 

I was a little bit worried to use high number of iterations because of old phones. Maybe raise to 10,000?

Reply:
Yes, but this second link is more recent (2017).

> For PBKDF2, the cost factor is an iteration count: the more times the PBKDF2 function is iterated, the longer it takes to compute the password hash. Therefore, the iteration count SHOULD be as large as verification server performance will allow, typically at least 10,000 iterations.

Reply:
I think we are good at 10,000. But we need to test on multiple devices and check performance.

Reply:
Testing on my computer it's really fast with 10,000 iterations. In the mobile app, with an iPhone 7, it takes ~3s to run the iterations. With the galaxy S8, takes ~1,5s.

With 1,000 iterations, takes ~1s with the iPhone7 and ~0,5s with the galaxy S8.

My suggestion is that we use 1,000 for now and, when we implement the security feature for the user decide how many iterations he wants, we set 1k for mobile and 10k for desktop as default.

What do you think?

Reply:
I agree. Can you create the card in our Trello, please?

Reply:
Done